moveit_ros/benchmarks/src/BenchmarkExecutor.cpp

```diff
@@ -532,6 +532,9 @@ bool BenchmarkExecutor::plannerConfigurationsExist(const std::map<std::string, s
     planning_interface::PlannerManagerPtr pm = planner_interfaces_[it->first];
     const planning_interface::PlannerConfigurationMap& config_map = pm->getPlannerConfigurations();
 
+    if (pm->getDescription().compare("stomp") || pm->getDescription().compare("chomp"))
```

Comment:
what is this for? add comment inline

Reply:
added comments explaining this, I put this here because for STOMP and CHOMP, the benchmarking was throwing an error as [this line](https://github.com/ros-planning/moveit/blob/kinetic-devel/moveit_ros/benchmarks/src/BenchmarkExecutor.cpp#L533) for CHOMP/STOMP returns a size of 0 for the config_map parameter which was causing an error. After putting this check, everything worked fine.